python/python/lance/dataset.py

```diff
@@ -1282,8 +1282,13 @@ def create_index(
         - **max_opq_iterations**: the maximum number of iterations for training OPQ.
         - **ivf_centroids**: K-mean centroids for IVF clustering.
 
-        If ``index_type`` is "DISKANN", then the following parameters are optional:
+        Optional parameters for "HNSW":
```

Comment:
This is `Optional parameters for IVF_HNSW_PQ` , right?

`HNSW` could be another implementation